src/site/content/en/blog/responsive-web-design-basics/index.md

```diff
@@ -0,0 +1,579 @@
+---
+title: Responsive Web Design Basics
+subhead: How to create sites which respond to the needs and capabilities of the device they are viewed on.
+description: |
+  How to create sites which respond to the needs and capabilities of the device they are viewed on.
+date: 2019-02-12
+updated: 2020-05-14
+authors:
+ - petelepage
+ - rachelandrew
+tags:
+ - blog
+ - css
+ - layout
+ - mobile
+ - ux
+draft: true
+---
+
+* [Set the viewport](#viewport)
```

Comment:
```suggestion
---

{# TODO(kayce): Remove this hard-coded ToC once #1983 lands. #}

* [Set the viewport](#viewport)
```
Thank you for hard-coding the ToC. I agree that it's needed and I've expressed to engineering that it's important that we get one across the site and they're working on it.

Reply:
@MichaelSolati is slated to work on it after he finishes the feedback widget.